resources/public/api/conf/1.0/application.yaml

```diff
@@ -30,6 +30,17 @@ components:
           scopes:
             write:self-assessment: Grant write access
             read:self-assessment: Grant read access
```

Comment:
We can probably delete the "User-Restricted" section here, since the Test API is not user-restricted.